src/main/resources/site/parts/contact/contact.es6

```diff
@@ -25,11 +25,13 @@ exports.preview = (req) => renderPart(req)
 function renderPart(req) {
   const WIDTH = 3 // how many boxes in a row
   const part = getComponent() || req
+  const page = getContent()
   const contactIdList = []
 
-  part.config.contacts = part.config.contacts && util.data.forceArray(part.config.contacts) || []
+  //checks page content for contacts first, then part for contacts and creates array
+  const contactsList = page.data.contacts.contact ? util.data.forceArray(page.data.contacts.contact) : part.config.contacts ? util.data.forceArray(part.config.contacts) : []
```

Comment:
We could avoid nesting ternaries, which makes for difficult reading. How about a simpler --

```
const contentContacts = ...
const partContacts = ...

const contactsList = [
  ...contentContacts,
  ...partContacts
]
```